seqr/views/apis/summary_data_api_tests.py

```diff
@@ -406,6 +406,14 @@ def test_saved_variants_page(self):
         if 'totalSampleCounts' in response_json:
             self.assertDictEqual(response_json['totalSampleCounts'], {'MITO': {'WES': 1}, 'SNV_INDEL': {'WES': 7}, 'SV': {'WES': 3, 'WGS': 3}})
 
+        all_tag_url = reverse(saved_variants_page, args=['ALL'])
+        response = self.client.get('{}?gene=ENSG00000135953'.format(all_tag_url))
+        self.assertEqual(response.status_code, 200)
+        report_variants = {'SV0027168_191912632_r0384_rare', 'SV0027167_191912633_r0384_rare'}
+        self.assertSetEqual(set(response.json()['savedVariantsByGuid'].keys()), {
+            'SV0000002_1248367227_r0390_100', 'SV0000013_prefix_19107_DEL_r00', *report_variants, *expected_variant_guids,
```

Comment:
this user ahas access to a saved SV with a key set, so moving this part of the test up means we actually test getting saved SVs